.github/workflows/scheduled.yml

```diff
@@ -117,10 +113,71 @@ jobs:
 
       - run: cargo test --release -p sncast
 
+  verify-version:
```

Comment:
Maybe it should be named something like `get_version`? From what I understand it just get's the version from `Cargo.toml`.